lib/assets/index.md

```diff
@@ -95,24 +94,20 @@ less critical than they originally thought.
 
 * **What Is Slack?**
   * Slack is a real-time chat system similar to IRC or Jabber. Here you can find
-Chef community members to ask questions, learn more about Chef, or socialize
+Habitat community members to ask questions, learn more about Habitat, or socialize
 with our awesome community.
 * **Why Slack?**
-  * Chef used IRC for our community chat for many years, however stagnation of the
-user experience for IRC combined with many improvements available from Slack
-prompted us to reconsider this. You can read more about the rationale of the
-move in [Chef RFC 74](https://github.com/chef/chef-rfc/blob/master/rfc074-community-slack.md).
+  * Stagnation of the user experience for IRC combined with many improvements available from Slack prompted us to avoid launching the Habitat Community on IRC. As Chef planned to move to Slack, we decided to _start_ on slack.
 * **Can I Use My Supermarket or Discourse Account?**
   * Unfortunately not at this time. For now your Slack account is independent of
-the other Chef community tools.
+the other Habitat or Chef community tools.
 * **Where I Can Find Logs Of Slack Channels?**
-  * You can find a permanent, searchable archive of our Slack channels at [SlackArchive](http://chefcommunity.slackarchive.io/).
+  * You can find a permanent, searchable archive of our Slack channels at [SlackArchive](http://habitat-sh.slackarchive.io/general/).
 You can also search the last ten thousand messages from within the Slack client. Be aware that archiving must be configured
 for each channel by inviting the `@slack_archive_bot` account.
-* **Where Else Can I Find Help With Chef?**
-  * We have a mailing list and forum at [discourse.chef.io](https://discourse.chef.io/)
-if you prefer a more asynchronous approach. We also have volunteers on
-[StackOverflow](https://stackoverflow.com/tags/chef). If volunteer support isn't
+* **Where Else Can I Find Help With Habitat?**
+  * We have a mailing list and forum at [forums.habitat.sh](https://forums.habitat.sh/)
```

Comment:
We don't need to say in Habitat-land that it's a mailing list -- that was for semantic "backwards compat" on the Chef Discourse. We really only have a forum :)